cupy/cuda/cudnn.pyx

```diff
@@ -679,6 +679,9 @@ cdef extern from 'cupy_cudnn.h' nogil:
     # Build-time version
     int CUDNN_VERSION
 
+    # Constants
+    float _CUDNN_BN_MIN_EPSILON 'CUDNN_BN_MIN_EPSILON'
```

Comment:
Should it not be a `double`?

Reply:
Thanks! It should be `double` as `epsilon` of `cudnnBatchNormalizationForwardTraining` etc. are typed as `double`.